react-exercise/src/pages/userProfile/UserProfile.scss

```diff
@@ -0,0 +1,116 @@
+.user-profile {
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    width: 100%;
+    background: var(--semantic-bg-primary, #FCFCFD);
+
+    .container {
+        display: flex;
+        flex-direction: column;
+        gap: 24px;
+        padding-top: 16px;
+        width: 1020px;
+        margin: 0 auto; // Center the container on the page 
+        
+        .back-to-search {
+            display: flex;
+            align-items: center;
+            text-decoration: none;
+            gap: 8px;
+            cursor: pointer;
+            width: 118px;
+            height: 18px;
+        
+            // text
+            color: var(--semantic-fg-secondary, #7A7C81);
+            font-family: "Brown LL";
+            font-size: 14px;
+            font-style: normal;
+            font-weight: 400;
+            line-height: 18px;
+            letter-spacing: -0.25px;
+          
+            .back-arrow-icon {
+                width: 16px;
+                height: 16px;
+            }
+        }
+    
+        .page-content {
+            color: var(--semantic-fg-neutral, #2C2D30);
+            border-radius: 32px;
+            background: var(--semantic-bg-secondary, #FFF);
+
+
+            font-family: "Brown LL";
+            font-size: 16px;
+            font-style: normal;
+            font-weight: 400;
+            line-height: 20px;
+            letter-spacing: -0.5px;
+
+            margin-top: 8px;
+            background: white;
+            display: flex;
+            flex-direction: column;
+            justify-content: space-between;
+
+            .profile-section {
+                display: flex;
+                flex-direction: column;
+                width: 1020px;
+                height: 679px;
+                padding: 32px 40px 0px 40px;
+                margin: 0 auto; // Center the profile section inside the container
+    
+                .user-preview {
+                    display: flex;
+                    flex-direction: row;
+                    gap: 24px;
+
+                    .user-details {
+                        display: flex;
+                        flex-direction: column;
+                        gap: 16px;
+                        width: 604px;
+
+                        .user-name {
+                            font-size: 32px;
+                            font-weight: 700;
+                            color: var(--semantic-text-primary, #2c2d30);
+                        }
+
+                        .change-details {
```

Comment:
I think you don't need this div, and can put the "justify-content: flex-end;" on the button itself

Reply:
Let me know if what I did is what you meant